solution/ui/prototype/src/components/header/SearchInput.vue

```diff
@@ -1,13 +1,10 @@
 <template>
-    <form
-        :class="formClasses"
-        action=""
-    >
-        <input
-            type="search"
-            name="q"
-            placeholder="Search Regulations"
-        />
+    <form :class="formClasses" action="">
+        <button class="search-chevron">
+            <p class="fa fa-chevron-down"></p>
+        </button>
+        <input type="search" name="q" placeholder="Search" />
```

Comment:
I think a v-text-field (https://vuetifyjs.com/en/api/v-text-field/#links)  with this icon might be simpler  https://materialdesignicons.com/icon/magnify

Reply:
I like the idea of a v-text-field but I think we should use the SVG icons from the design.  You can select them and then right click on them and choose "Copy/Paste as..." > "Copy as SVG", which you can use to create your own inline icon components.  I just learned about this earlier this morning:

https://v2.vuejs.org/v2/cookbook/editable-svg-icons.html

You can wrap SVG markup in a template and then save it as a Vue component.

Vuetify then lets you register these icon components in the Vuetify config to be used in the append-icon/prepend-icon/etc slots:

https://stackoverflow.com/questions/51614557/how-to-add-custom-svg-icon-in-vuetify-vue

I hacked on this branch a bit this morning and ended up with this:

https://user-images.githubusercontent.com/1651907/155004503-33e9b87f-91c7-4597-a2af-23acfa074972.mov

which gets us a lot of the way there functionality-wise.